src/pudl/metadata/resources/eia861.py

```diff
@@ -74,7 +74,18 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_demand_response": {
```

Comment:
@zaneselvans we should rename them if we think the names don't accurately describe the contents of the table. However, IDK if we have the hours for it this quarter. 

I don't think it'd be the worst thing to do a data release with the current names and then update them in the future. The EIA 861 tables require more modeling/cleaning work, correct? Maybe we can update the names then?

Reply:
Yes, they'll definitely require more work if we want them to be comparable in normalization and consistency to the other EIA data. I think there are a handful of obvious changes here that will probably not be disruptive and will make the table names meaningful.  Let's defer anything that's not obvious. I don't think we can really get away from the fact that tables and columns will change over time, but I don't know how to gracefully deal with that kind of schema drift either, beyond offering stable releases so folks can choose when to migrate.

Reply:
Ok! After looking at the table names more closely, I agree there are some easy fixes we can tackle in this PR.

Reply:
Ok! I'll tackle the suggested table name changes. I'm sure there is a lot of nit-picking we can do to these tables at a later time.

Reply:
On second thought, I kind of agree with @bendnorman here. Renaming tables opens a can of worms and makes it really hard not to dig into the content of the tables more meaningfully. I think we are due for an 861 revamp, but it deserves a bit more time/effort, and I don't think it makes sense to dedicate any time to doing that until we can do it in full.

Reply:
Using #3301 to collect table name and structure change information.